CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## Unreleased
+## v0.69.0
+
+Updates smartagent, javaagent, and opentelemetry to `v5.27.2`, `1.20.0`, and `v0.69.0`/`v0.69.1` respectively.
```

Comment:
For consistency:
- The otel note should match previous releases
- Move the smart agent and java agent notes to a "Enhancements" section
- Include links to all respective github releases for each version